Github changes plugin used in release script does not use two-factor authentication

```
[INFO] --- github-changes-maven-plugin:1.0:github-changes (default-cli) @ 
adam-parent-spark2_2.11 ---
[INFO] Writing milestone 24 changes to temporary file
[INFO] Retrieving issues for milestone 24 from repository bigdatagenomics/adam
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for ADAM_2.11 0.30.0-SNAPSHOT:
[INFO]
[INFO] ADAM_2.11 .......................................... FAILURE [  8.135 s]
[INFO] ADAM_2.11: Shader workaround ....................... SKIPPED
[INFO] ADAM_2.11: Avro-to-Dataset codegen utils ........... SKIPPED
[INFO] ADAM_2.11: Core .................................... SKIPPED
[INFO] ADAM_2.11: APIs for Java, Python ................... SKIPPED
[INFO] ADAM_2.11: CLI ..................................... SKIPPED
[INFO] ADAM_2.11: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.627 s
[INFO] Finished at: 2019-12-10T16:56:39-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.heuermh.maven.plugin.changes:
github-changes-maven-plugin:1.0:github-changes (default-cli) on project
adam-parent-spark2_2.11: could not write changes for milestone id 24, caught
{"message":"Must specify two-factor authentication OTP code.","documentation_url":
"https://developer.github.com/v3/auth#working-with-two-factor-authentication"}:
Server returned HTTP response code: 401, message: 'Unauthorized' for URL:
https://api.github.com/repos/bigdatagenomics/adam: Server returned HTTP
response code: 401 for URL: https://api.github.com/repos/bigdatagenomics/adam
-> [Help 1]
```

--- a/scripts/release/release.sh
+++ b/scripts/release/release.sh
@@ -17,7 +17,7 @@ milestone=$3
 branch=$(git status -bs | awk '{ print $2 }' | awk -F'.' '{ print $1 }' | head -n 1)
 
 # update changelog per Github milestone
-mvn com.github.heuermh.maven.plugin.changes:github-changes-maven-plugin:1.0:github-changes -DmilestoneId=${milestone}
+mvn com.github.heuermh.maven.plugin.changes:github-changes-maven-plugin:1.1:github-changes -DmilestoneId=${milestone}
 git commit -a -m "Modifying changelog."
 
 # update R version

